modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss

```diff
@@ -53,90 +19,148 @@ $tab-header-height: 38px;
 		}
 	}
 
-	.tab-pane {
-		max-height: calc(100vh - #{$control-menu-height + $tab-header-height});
-		overflow-y: auto;
+	.modal-info .btn-monospaced {
+		color: $secondary;
 	}
 }
 
-.applications-menu-nav {
-	margin: 0;
-	padding: 0.2rem;
+.applications-menu-header .nav-link {
+	color: $secondary;
+	padding-bottom: 1rem;
+	padding-top: 1rem;
 
-	@include media-breakpoint-down(xs) {
-		@at-root {
-			.mobile #{&} {
-				padding-bottom: 7rem;
-			}
-		}
+	&.active {
+		color: $dark;
 	}
+}
 
-	@include media-breakpoint-up(sm) {
-		padding: 1.5rem 2.5rem 1.5rem 1.75rem;
-	}
+.applications-menu-wrapper {
+	display: flex;
+	flex-direction: column;
+	max-height: 100vh;
+}
 
-	.applications-menu-nav-header {
-		color: $secondary;
-		font-size: 0.875rem;
-		font-weight: 700;
-		line-height: 3rem;
-		text-transform: uppercase;
+.applications-menu-bg {
+	@include media-breakpoint-up(md) {
+		background: linear-gradient(
+			to right,
+			$white 0%,
+			$white 50%,
+			$light 51%,
+			$light 100%
+		);
+
+		.container-fluid {
+			background-color: $white;
+		}
 	}
+}
```

Comment:
it doesn't change anything but if we could change this following a "class first pattern"
```scss
.applications-menu-bg {
  @include media-breakpoint-up(md) {
    background: linear-gradient(
      to right,
      $white 0%,
      $white 50%,
      $light 51%,
      $light 100%
    );
  }

  .container-fluid {
    @include media-breakpoint-up(md) {
      background-color: $white;
    }
  }
}
```